h/accounts/test/views_test.py

```diff
@@ -413,7 +420,7 @@ def test_forgot_password_redirects_on_success(authn_policy):
 
 @pytest.mark.usefixtures('routes_mapper')
 def test_forgot_password_form_redirects_when_logged_in(authn_policy):
-    request = DummyRequest()
+    request = mock.Mock(method="POST", authenticated_userid="jane")
```

Comment:
You now have two lines next to one another saying different things: `authenticated_userid="jane"` and `authn_policy.authenticated_userid.return_value = "acct:jane@doe.org"`.